Python/Source/RandomPick.cpp

```diff
@@ -7,26 +7,19 @@ using namespace pybind11;
 using namespace Rover;
 
 namespace {
-  class PythonRandomPick {
+  class IntegerRange : public Range<std::size_t, std::size_t> {
```

Comment:
I still had to do this export to make it work through `PythonPick`. Let me know if you prefer this removed while `PythonRandomPick` no longer uses `PythonPick`.
The inheritance is to prevent export clashes in case a user exports `Range<std::size_t, std::size_t>` on their own.

Reply:
I would prefer this be removed and you create take a ```Range(...)``` and wrap it in a ```Box<object>``` and use that as the ```choice``` for ```PythonPick```.